src/Agent.Worker/Agent.Worker.csproj

```diff
@@ -9,6 +9,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\Microsoft.VisualStudio.Services.Agent\Microsoft.VisualStudio.Services.Agent.csproj" />
+    <ProjectReference Include="..\Agent.Listener\Agent.Listener.csproj" />
```

Comment:
why you need this?

Reply:
We added a feature to write Capabilities to disk too. The ICapabilitiesManager exists in the Listener.